tensorflow/c/c_api.cc

```diff
@@ -1344,7 +1344,8 @@ void TF_OperationGetAttrString(TF_Operation* oper, const char* attr_name,
         InvalidArgument("Attribute '", attr_name, "' is not a string");
     return;
   }
-  if (max_length <= 0) {
+  if (max_length == 0) {
+    status->status = InvalidArgument("Attribute '", max_length, "' is zero");
```

Comment:
I don't think this is correct.  `max_length` is the length `value` and so if it is `0` then there is nothing to do here.

Reply:
Agree with your point. I checked its usage and its only being called in two places in file: tensorflow/c/c_api_test.cc . The max_len is just the length of the value and its 0 if the value is a nullptr. My intent was to only remove the check for unsigned value being less than zero but it seems to be unnecessary here.